src/H5C.c

```diff
@@ -103,6 +103,9 @@
 /* Local Typedefs */
 /******************/
 
+/* Alias for pointer to cache entry, for use when allocating sequences of them */
+typedef H5C_cache_entry_t *H5C_cache_entry_ptr_t;
```

Comment:
Changes in this file are to align with d5130bb57332ce0049302c5836bbf666b23513cd from develop